scripts/rpmbuilder.Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG RPM_VER
+FROM quay.io/freedomofpress/rpmbuilder:${RPM_VER}
```

Comment:
Could we also pin the hash of the image?

Reply:
:+1: